common/src/app/common/logic/variants.cljc

```diff
@@ -60,6 +60,11 @@
                                                :value v}) remaining)]
     (into assigned new-properties)))
 
+(defn- dashes-to-end
+  [data]
```

Comment:
This parameter should have a meaningful name. In general, "data" is a too abstract name. In Penpot we have adopted the convention of using this name for a file-data object (what it's not the case here).

Perhaps it could be "values" or "property-values".

Reply:
Fixed